DOCSP-36869 compass auto updates (#607)
mongodb/docs-compass: source/upgrade.txt

   :figwidth: 600px
   :alt: Privacy settings

> If you enable auto updates and have not downgraded |compass|,

Reviewer: ```suggestion
If you enable auto updates and don't downgrade |compass|,
```

Author: changed to "didn't downgrade" to capture past tense

|compass-short| automatically updates the application four hours after

> MongoDB releases new versions. The update does not require your explicit

Reviewer 2: I wouldn't mention a specific time frame here (and the 4 hours thing is not really accurate anyway)

Author: Thanks for this clarification! I removed the mentions of 4 hours

confirmation. After your machine finishes the update, |compass-short|

> shows a toast suggesting that you restart the application to apply the

Reviewer 2: ```suggestion
shows a notification suggesting that you restart the application to apply the
```

I don't know if users know what a toast is, tbh.

Author: Changed to "notification"!

apply the update. |compass-short| does not apply the version update
until you restart the application. 

> If you enable auto updates and have downgraded |compass| (you installed
> |compass| and then manually installed an earlier version),

Reviewer: ```suggestion
If you enable auto updates and subsequently downgrade |compass| by
manually installing an earlier version,
```

|compass-short| does not automatically update when MongoDB releases a
new version. If a newer version is available, |compass-short| asks you
to confirm that you want to install the update through a notification.

> You can dismiss the notification and refrain from updating. 

Reviewer: ```suggestion
You can dismiss the notification to avoid updating. 
```

If a new update is available and you want to update |compass-short|

> prior to the four hour delay, you can click :guilabel:`Check for

Reviewer 2: Similarly here, the 4 hour things is a different timer and I just wouldn't mention it at all.

> If a new update is available and you want to update |compass-short|
> before the automatic update, you can click :guilabel:`Check for
> updates` in the :guilabel:`MongoDB Compass` system menu. 

Reviewer: ```suggestion
To update |compass-short| before the next automatic update, click 
:guilabel:`Check for updates` in the :guilabel:`MongoDB Compass` 
system menu:
```

.. figure:: /images/compass/check_for_updates.png
  :scale: 60%
  :alt: Check for updates location under the MongoDB Compass system menu

> If you click on :guilabel:`Check for updates` and an update is
> available, |compass-short| shows a notification that asks if you want to

Reviewer: ```suggestion
If an update is
available, |compass-short| shows a notification that asks if you want to
```

install and restart the application immediately. 

> If an update is in progress, |compass-short| disables the

Reviewer: ```suggestion
While an update is in progress, |compass-short| disables the
```
